<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>usb2CanDataAnalTool</class>
 <widget class="QDialog" name="usb2CanDataAnalTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>625</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>mainlog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="pushButton_closeWin">
   <property name="geometry">
    <rect>
     <x>808</x>
     <y>576</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Title">
   <property name="geometry">
    <rect>
     <x>416</x>
     <y>8</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>USB2CAN 数据分析工具</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_FileInfo">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>48</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_DataTable">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>160</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_Widget">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>184</y>
     <width>881</width>
     <height>377</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>64</y>
     <width>633</width>
     <height>89</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QLabel" name="label_Can1">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_Time">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_Date">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_Can2">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_Name">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_dataNum">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="5" margin="10"/>
 <includes>
  <include location="local">这是一个用于分析USBCAN2.0数据的程序</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_closeWin</sender>
   <signal>released()</signal>
   <receiver>usb2CanDataAnalTool</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>765</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
